### PR TITLE
Fix deprecations on Buffer and Crypto

### LINKED ISSUE #14: Deprecations
```
(node:1133) [DEP0091] DeprecationWarning: crypto.DEFAULT_ENCODING is deprecated.
(node:1133) [DEP0010] DeprecationWarning: crypto.createCredentials is deprecated. Use tls.createSecureContext instead.
(node:1133) [DEP0011] DeprecationWarning: crypto.Credentials is deprecated. Use tls.SecureContext instead.[root]
```

Are you aware of this?

### PR DESCRIPTION
**Pull request by murbanowicz**
Fixes #14

### CODE REVIEW

**Comment by Levino** [comment]
`package.json` and `yarn.lock` should be updated too, right_

**Comment by murbanowicz** [comment]
Updated `package.json` however I don't see any changes in `yarn.lock`

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "mock-jwks",
   "description": "A tool to mock a JWKS for development of microservices who consume jwts signed with RSA",
-  "version": "0.1.8",
+  "version": "0.1.9",
```
Comment (Levino) [inline package.json]: Please do not change the version. I do this when releasing.

```suggestion
  "version": "0.1.8",
```

**Comment by Levino** [comment]
When you force push you must request a review by me. If not I will not get a notification.

**Comment by murbanowicz** [comment]
Ok.
I think we can switch to 10.15.3 on CI as it is LTS at this moment so it is wise to be on LTS. What you think? I am testing using LTS locally as always.

**Comment by murbanowicz** [comment]
Also, I would encourage you to use alpine images on CI as much as possible as it speeds up the pipelines.

**Comment by Levino** [comment]
Okay. Change it to alpine.

**Comment by Levino** [comment]
I also created https://github.com/Levino/mock-jwks/issues/16 fyi.

**Comment by Levino** [comment]
misclick

**Comment by Levino** [comment]
Replaced by #17

**Pull request closed without merging**